docs/src/alm.md

```diff
@@ -118,16 +112,75 @@ pixwin
 gaussbeam
 ```
 
-## Multiplying a set of Alm by a generic function of $\ell$
+## Algebraic operations in harmonic space
+
+Healpix.jl provides overloads of the `Base` functions `+`, `-`, `*`, `/` and `\` (inverse division),
+as well as `LinearAlgebra.dot`, allowing to carry out these fundamental operations
+element-wise in harmonic space directly.
+
+For example, an element-wise sum between two `Alm` objects can be performed as follows:
+
+```julia
+using Healpix
+
+#just two constant Alm objects
+myalm1 = Healpix.Alm(5,5, ones(ComplexF64, Healpix.numberOfAlms(5)))
+myalm1 = Healpix.Alm(5,5, ones(ComplexF64, Healpix.numberOfAlms(5)))
+
+alm_sum = myalm1 + myalm2 #each element will be = 2 + 0im
+```
+
+#### Multiplying or dividing a set of Alm by a generic function of $\ell$ or a constant
+
+The operators `*` and `/` (or `\`) can be used to multiply or divide an `Alm` by an $\ell$-dependent
+generic function $f_\ell$ (or just a constant, of type `Number`).
 
-You can use the function [`almxfl`](@ref) (or [`almxfl!`](@ref)) to multiply (in-place)
- a set of $a_{\ell m}$ coefficients by an $\ell$-dependent generic function $f_\ell$.
+Depending on which order the arguments are placed, the result will be overwritten
+in place or returned as a new instance of `Alm` type:
+
+```julia
+using Healpix
+
+#just two constant Alm objects
+myalm = Healpix.Alm(5,5, ones(ComplexF64, Healpix.numberOfAlms(5)))
+myf_l = ones(Healpix.numberOfAlms(5)) .* 2
+
+#will return a new object:
+myalm*myf_l
+myalm/myf_l
+
+#will overwrite myalm:
+myf_l*myalm
+myf_l\myalm #here the inverse division must be used
+```
+
+In either case the call to such operator consists in a shortcut to the function [`almxfl`](@ref) (or [`almxfl!`](@ref), if in place).
```

Comment:
This is counter-intuitive and hard to remember; better to remove in-place operators and force people to use explicit functions instead. (This is also suggested in the [Julia forum](https://discourse.julialang.org/t/overload-addition-assignment-operator/28836/13?u=maurizio_tomasi).)